maestro-driver-ios/maestro-driver-iosUITests/Routes/Route.swift

```diff
@@ -1,9 +1,34 @@
-enum Route: String {
+import FlyingFox
+
+enum Route: String, CaseIterable {
     case subTree = "/subTree?appId=*"
     case runningApp = "/runningApp"
     case swipe = "/swipe?appId=*"
     case inputText = "/inputText?appId=*"
     case touch = "/touch?appId=*"
```

Comment:
The `?appId=*` can be dropped. The handlers should emit an error if the parameter is not present, while currently a 404 will be emitted if the parameter is not present.
Dropping the `?appId=*` allows us to remove the string literals altogether here.